src/pages/api/returnValidity.ts

```diff
@@ -0,0 +1,133 @@
+import { NextApiResponse } from 'next';
+import * as yup from 'yup';
+import { redirectToError, redirectTo } from './apiUtils/index';
+import { isSessionValid } from './apiUtils/validator';
+import { ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { RETURN_VALIDITY_ATTRIBUTE } from '../../constants';
+
+const radioButtonError = 'Choose one of the options below';
+const inputError = 'Enter a whole number';
+const daysAmountError = 'Enter a number of days between 1 and 31';
+const weeksAmountError = 'Enter a number of weeks between 1 and 52';
+const monthsAmountError = 'Enter a number of months between 1 and 72';
+const yearsAmountError = 'Enter a number of years greater than 1';
+const selectError = 'Choose one of the options from the dropdown list';
+
+export const returnValiditySchema = yup
```

Comment:
Upon testing this, we error if the user inputs a valid number, and follows it with a space. Should we trim the string to remove leading and trailing whitespace?